packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -242,6 +265,7 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
           directionalHintFixed={ directionalHintFixed }
         >
           <div style={ contextMenuStyle } ref={ (host: HTMLDivElement) => this._host = host } id={ id } className={ css('ms-ContextualMenu-container', className) }>
```

Comment:
Suggestion, use `warnMutuallyExclusive` for warning about mutually exclusive props.

Reply:
Which props would need this warning?

Reply:
After talking to David offline, it looks like this is unnecessary.